src/PrestaShopBundle/Form/Admin/Product/ProductCombination.php

```diff
@@ -140,6 +140,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 new Assert\Type(array('type' => 'numeric')),
             )
         ))
+        ->add('attribute_low_stock', 'Symfony\Component\Form\Extension\Core\Type\NumberType', array(
+            'label' => $this->translator->trans('Low stock level', [], 'Admin.Catalog.Feature'),
```

Comment:
Son, you disappoint me.

`[]` => `array()`

Reply:
I just copy and past... f*************k!!